core/metacling/src/TClingCallFunc.h

```diff
@@ -142,22 +142,21 @@ class TClingCallFunc {
    ~TClingCallFunc() = default;
 
    explicit TClingCallFunc(cling::Interpreter *interp)
-      : fInterp(interp), fWrapper(0), fIgnoreExtraArgs(false), fReturnIsRecordType(false)
```

Comment:
The commit message does not seem to correspond to the code change. I would have expected
```
fReturnIsRecordType is now unused, remove it.
```

Reply:
Fixed.